data/news/platform/0install-intro.md

```diff
@@ -30,12 +30,12 @@ You can install a binary release of OPAM on most systems by first installing the
 
 2. Install OPAM:
 
-        $ 0install add opam http://tools.ocaml.org/opam.xml
+        $ 0install add opam https://tools.ocaml.org/opam.xml
```

Comment:
Note that the HTTPS version of https://tools.ocaml.org/opam.xml is trying to load its CSS using HTTP, which is blocked by navigators, and it's causing the site to render without the style.